src/Workspaces/Core/Portable/FindSymbols/Shared/AbstractSyntaxIndex_Persistence.cs

```diff
@@ -116,10 +116,11 @@ internal partial class AbstractSyntaxIndex<TIndex>
 
             var directivesChecksum = s_ppDirectivesToChecksum.GetValue(
                 project.ParseOptions!,
-                static parseOptions => Checksum.Create(parseOptions.PreprocessorSymbolNames));
+                static parseOptions =>
+                    new StrongBox<Checksum>(CodeAnalysis.Checksum.Create(parseOptions.PreprocessorSymbolNames)));
```

Comment:
>StrongBox [](http://example.com/codeflow?start=24&length=9)

Curious what advantages using StrongBox gives here. Checksum doesn't really have state, and it's not much bigger than a pointer for copying size comparison, but maybe there some other advantage to using it I'm not yet aware of.

Reply:
StrongBox is required here as CWT doesn't allow struct values.